psydac/api/grid.py

```diff
@@ -98,9 +98,10 @@ def axis(self):
     @property
     def ext(self):
         return self._ext
+
 #==============================================================================
 class BasisValues():
-    def __init__( self, V, grid, nderiv , ext=None):
+    def __init__( self, V, grid, nderiv , trial=False, ext=None):
```

Comment:
Can you please add a docstring to this class? I think it deserves one now

Reply:
Thanks for adding the docstring. But the constructor's docstring should be placed outside of the `__init__` function. In other words, this should be the class' docstring.

Reply:
done